app/assets/stylesheets/styling_overrides.scss

```diff
@@ -0,0 +1,32 @@
+
+$red: #940926;
+$dark-red: darken( $red, 20% );
+$lt-gold: #d8c9a1;
+$font-family-sans-serif: Arial, "Helvetica Neue", sans-serif;
+
+#background {
+  background: url(/assets/Boston_Skyline_Night_BW.png) no-repeat fixed center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100%;
+  padding-top: 59.05%;
+}
+
+.index-boxes {
+  background-color: $red;
+  border: 2px solid $dark-red;
+  color: $lt-gold;
+  height: 175px;
```

Comment:
Properties should be ordered background-color, border, color, font-family, font-size, height, margin